hcipy/aperture/realistic.py

```diff
@@ -349,6 +349,94 @@ def aper(grid):
 
 	return aper
 
+def make_luvoir_b_aperture(normalized=False, with_segment_gaps=True, gap_padding=1, segment_transmissions=1,
+						   return_header=False, return_segments=False):
+	'''
```

Comment:
First line of documentation should contain a one-line description of the function. See `luvoir_a` above. Further description follows below with a black line in between. Conforming to numpydoc (https://numpydoc.readthedocs.io/en/latest/format.html#short-summary).